include/ensmallen_bits/sgd/update_policies/weight_decay_momentum_update.hpp

```diff
@@ -0,0 +1,101 @@
+/**
+ * @file weight_decay_momentum_update.hpp
+ * @author Ajinkya Tejankar
+ * @author Niteya Shah
+ * Momentum update with weight decay for Stochastic Gradient Descent
+ *
+ * ensmallen is free software; you may redistribute it and/or modify it under
+ * the terms of the 3-clause BSD license.  You should have received a copy of
+ * the 3-clause BSD license along with ensmallen.  If not, see
+ * http://www.opensource.org/licenses/BSD-3-Clause for more information.
+ */
+#ifndef ENSMALLEN_WEIGHT_DECAY_MOMENTUM_UPDATE_HPP
+#define ENSMALLEN_WEIGHT_DECAY_MOMENTUM_UPDATE_HPP
+
+namespace ens {
+
+/**
+ * De-Coupled Weight Decay Policy for SGD with Momentum (SGDW).
+ *
+ * This Implments Decoupled Weight Decay Policy  in which the weight decay is
```

Comment:
Can you fix this comment please?  It's not clear why some of these words are capitalized.